### PR TITLE
Unit test for multisig transaction - Subtask #456 - Closes #832

### LINKED ISSUE #832: Add unit tests for multisignature transactions
Parent issue: #456 

More unit tests are needed for this section of the code.

 https://github.com/LiskHQ/lisk/blob/development/logic/multisignature.js

### PR DESCRIPTION
**Pull request by SargeKhan**
Closes #832

### CODE REVIEW

File: logic/multisignature.js [hunk 1]
```diff
@@ -23,14 +23,15 @@ __private.unconfirmedSignatures = {};
  * @param {Object} logger
  */
 // Constructor
-function Multisignature (schema, network, transaction, logger) {
+function Multisignature (schema, network, transaction, account, logger) {
```
Comment (4miners) [inline logic/multisignature.js]: Reason of all changes in this file?

File: test/unit/logic/multisignature.js [hunk 2]
```diff
@@ -49,46 +53,566 @@ var multiSigAccount2 = {
...
+				trs.asset.multisignature.keysgroup = [];
+
+				multisignature.verify(trs, sender, function (err) {
+					expect(err).to.equal('Invalid multisignature keysgroup. Must not be empty');
+					done();
+				});
+			});
+		});
+
+		describe('when min <= 1', function () {
```
Comment (4miners) [inline test/unit/logic/multisignature.js]: Imo we should test also first and last valid path of a range - so 2 and 15 here.

File: logic/multisignature.js (hunk 1 above)
Comment (MaciejBaj) [reply logic/multisignature.js]: `library.logic.account` is being used in code

File: test/unit/logic/multisignature.js (hunk 2 above)
Comment (SargeKhan) [reply test/unit/logic/multisignature.js]: Added tests for these values

**Pull request merged**